Documentation/NetworkBoot.md

```diff
@@ -150,45 +188,3 @@ echo 0 > /sys/devices/virtual/net/br0/bridge/multicast_querier
 qemu-system-i386 -m 4096 -netdev tap,ifname=tap0,script=no,downscript=no,id=network0 -device e1000,netdev=network0 -boot n -debugcon stdio -s
 ```
 
-## SerenityOS network booting via USB (iPXE)
```

Comment:
I'd not remove the instructions for iPXE as it's technically still a network boot option, just not the conventional one (so you don't really use a PXE option here).
Maybe a different document for this is more appropriate.

Reply:
Also, if we still want to document how to work with ipxe then the new script is:
```
#!ipxe
dhcp
kernel http://X.Y.Z.W/Prekernel serial_debug root=/dev/ramdisk0
module http://X.Y.Z.W/Kernel.drow
module http://X.Y.Z.W/_disk_image
boot
```
I confirmed it worked on QEMU. It probably will work on my ICH7 test machine too.

Reply:
I rearranged the guide to follow the actual process of network booting (PXE firmware -> DHCP/TFTP server -> bootloader), so that section has been moved to the `Bootloaders > iPXE` section. I've also simplified all samples by assuming a TFTP server has been set up and the DHCP server points to it, otherwise this guide will be way too complex. Seasoned sysadmins can tune their setup.